src/ops/spec_inc_multihead_self_attention.cc

```diff
@@ -127,7 +127,7 @@ Tensor
   }
   if (bias) {
     // q, k, v, o
-    int dims[1] = {embed_dim * 4};
+    int dims[1] = {(qProjSize + kProjSize + vProjSize) * num_heads + oProjSize};
```

Comment:
Hi Gabriele, just to make sure, do we have any situation that the `embed_dim * 4` is not equal to `qProjSize + kProjSize + vProjSize) * num_heads + oProjSize`?

Reply:
In Megatron-LM's tensor model parallelism, we partition attention heads across GPUs, so `num_heads` of each GPU 
can be smaller than the total number of attention heads

Reply:
In the current model we have, the two quantities are indeed always identical. I changed this just to make it easier to remember the data layout

Reply:
thanks!